### ISSUE TITLE
Pre-trained model or Datasets？

### ISSUE THREAD

**Issue by PasserJW767** [Yushi-Du/PartMotion#2]
Hi, 

Thanks for your grateful work! I tried running your code on my dataset, but I found that the loss fails to converge. I'm not sure if it's due to my dataset. Would it be presumptuous of me to request that you make your dataset or pre-trained model public?

Many thanks! Looking forward to your reply.
